### PR TITLE
Clarify max upload size settings for Cloud users

### PR DESCRIPTION
**Pull request by skttl**
I was confused to why Cloud needs a web.config to set a max upload size, so I asked in Discord.

https://discordapp.com/channels/869656431308189746/882984067312787466/1064481265279049818

### CODE REVIEW

File: umbraco-cms/reference/configuration/maximumuploadsizesettings.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ To configure the default 28.6MB upload limit using IIS, we have to create a web.
 `maxAllowedContentLength` is specified in bytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes
 
 {% hint style="info" %}
-If your site is hosted on Umbraco Cloud, this is the method you should use.
+If your site is hosted on Umbraco Cloud, this is the method you should use. Umbraco Cloud uses IIS for hosting, and IIS needs the setting to be in a web.config file for this to work.
```
Comment (sofietoft) [inline umbraco-cms/reference/configuration/maximumuploadsizesettings.md:31]:
```suggestion
**Are you hosting your site on Umbraco?**
Umbraco Cloud uses IIS for hosting. This means you need to add the setting in a `web.config` file for this to work on your Umbraco Cloud hosted sites.
```

**Review by sofietoft — COMMENTED** [review]
Thanks a bunch for clarifying this @skttl ! 💪 MUCH appreciated.

I just made a few changes to highlight the note a little more.
Let me know what you think 🙏

File: umbraco-cms/reference/configuration/maximumuploadsizesettings.md (hunk 1 above)
Comment (skttl) [reply umbraco-cms/reference/configuration/maximumuploadsizesettings.md:31]: I guess it should be `**Are you hosting our site on Umbraco Cloud?**`

Comment (sofietoft) [reply umbraco-cms/reference/configuration/maximumuploadsizesettings.md:31]: 😂 Oh my... Yes. that's a "typo" on my end. Thanks for keeping me on my toes 😄 I'll correct it!

**Comment by skttl** [comment]
All good :)

**Pull request merged**